modules/Students/student_view_details_notes_add.php

```diff
@@ -81,12 +81,22 @@
 				print "</div>" ;
 			} 
 			
-
-			if ($_GET["search"]!="") {
-				print "<div class='linkTop'>" ;
-					print "<a href='" . $_SESSION[$guid]["absoluteURL"] . "/index.php?q=/modules/Students/student_view_details.php&gibbonPersonID=$gibbonPersonID&search=" . $_GET["search"] . "&subpage=$subpage&category=" . $_GET["category"] . "'>" . _('Back to Search Results') . "</a>" ;
-				print "</div>" ;
+			$search="";
+			if (isset($_GET["search"])) {
+				$search=$_GET["search"] ;
+			}
+			$allStudents="" ;
+			if (isset($_GET["allStudents"])) {
+				$allStudents=$_GET["allStudents"] ;
 			}
+			$sort="surname, preferredName";
+			if(isset($_GET["sort"])) {
+				$sort=$_GET["sort"];
+			}
+
+			print "<div class='linkTop'>" ;
```

Comment:
As per previous comment, we do want to check for search or sort, and the display if one, the other or both are set to something.